hawc/apps/epi/api.py

```diff
@@ -87,37 +100,425 @@ def result_heatmap(self, request, pk):
         return Response(export)
 
 
-class StudyPopulation(AssessmentViewset):
+class Criteria(PermCheckerMixin, AssessmentEditViewset):
+    perm_checker_key = "assessment"
+    assessment_filter_args = "assessment"
+    model = models.Criteria
+    serializer_class = serializers.CriteriaSerializer
+
+
+class StudyPopulation(PermCheckerMixin, viewsets.ModelViewSet):
```

Comment:
why can't we use an AssessmentViewset or AssessmentEditViewset - this adds all the permission checking built-in. 

Same with other Viewsets - I'd rather not use generic DRF stuff again.